examples/borer-circe-migration/src/main/scala/btest/Transcoding.scala

```diff
@@ -0,0 +1,46 @@
+package pl.marconzet
+package btest
+
+import io.bullet.borer.Dom.ArrayElem.Unsized
+import io.bullet.borer.{Cbor, Codec, Dom, Json}
+import io.bullet.borer.derivation.MapBasedCodecs._
+
+object Transcoding extends App {
+  case class Employee1(nick: String, age: Int)
+
+  case class Employee2(nick: String, age: Option[Int])
+
+  implicit val employee1Codec: Codec[Employee1] = deriveCodec[Employee1]
+  implicit val employee2Codec: Codec[Employee2] = deriveCodec[Employee2]
+
+  val value = Employee2("Alice", None)
```

Comment:
Hmmm okay so this naming here suggests that you're transcoding from an **newer model to an older one**, right? Will this be the more common scenario, or rather will transcoding from **older to newer** model be more common?